docs/docusaurus/docs/components/setup/link_lists/_data_context_initialize_instatiate_save.md

```diff
@@ -6,14 +6,14 @@
 </summary>
 
 **Quickstart Data Context**
-- [Instantiate a Data Context](/docs/oss/guides/setup/configuring_data_contexts/instantiating_data_contexts/instantiate_data_context)
+- [Instantiate a Data Context](/oss/guides/setup/configuring_data_contexts/instantiating_data_contexts/instantiate_data_context.md)
 
 **Filesystem Data Contexts**
-- [How to initialize a filesystem Data Context in Python](/docs/oss/guides/setup/configuring_data_contexts/instantiating_data_contexts/instantiate_data_context)
-- [How to instantiate a specific Filesystem Data Context](/docs/oss/guides/setup/configuring_data_contexts/instantiating_data_contexts/instantiate_data_context)
+- [How to initialize a filesystem Data Context in Python](/oss/guides/setup/configuring_data_contexts/instantiating_data_contexts/instantiate_data_context.md)
```

Comment:
@Rachel-Reverie I think this one is wrong, but I was struggling to figure out the right path. Can you point me in the right direction?